docs/bin/plugin_formatter.py

```diff
@@ -117,7 +117,7 @@ def write_data(text, options, outputname, module):
         fname = os.path.join(options.output_dir, outputname % module)
         fname = fname.replace(".py","")
         f = open(fname, 'w')
-        f.write(text.encode('utf-8'))
+        f.write(to_text(text))
         f.close()
```

Comment:
File operations are better handled as:
``` python
f = open(fname, 'wb')
f.write(to_bytes(text))
```

Sending text to the file makes python3 implicitly encode to bytes and whenever python encodes implicitly it can traceback if the data and encoding are incompatible.